scheduler/src/cook/kubernetes/api.clj

```diff
@@ -777,15 +785,19 @@
       (assert (= pod-name-from-pod pod-name)
               (str "Pod name from pod (" pod-name-from-pod ") "
                    "does not match pod name argument (" pod-name ")"))
-      (log/info "Launching pod with name" pod-name "in namespace" namespace ":" (Yaml/dump pod))
+      (log/info "In" compute-cluster-name "launching pod with name" pod-name "in namespace" namespace ":" (Yaml/dump pod))
```

Comment:
Same comment as above about the pattern for these logs.

Reply:
Changed.